README.md

```diff
@@ -31,17 +32,16 @@ OpenNext aims to support all Next.js 14 features. Some features are work in prog
 - [x] Server-side rendering (SSR)
 - [x] Incremental static regeneration (ISR)
 - [x] Middleware
+- [x] Server actions
 - [x] Image optimization
 - [x] [NextAuth.js](https://next-auth.js.org)
 - [x] Running at edge
-- [x] No cold start
+- [x] Almost no coldstart
```

Comment:
We should add an asterisk to provide more details.
eg, on Lambda, there will always be a scenario where a Lambda is created that is > warmed instance, so that request will be cold. For Nextjs, not all routes are warmed from the warm event.